fastapi_backend/app/main.py

```diff
@@ -54,3 +59,46 @@
 @app.get("/authenticated-route", tags=["custom-auth"])
 async def authenticated_route(user: User = Depends(current_active_user)):
     return {"message": f"Hello {user.email}!"}
+
+
+@app.get("/items/", tags=["item"], response_model=list[ItemRead])
```

Comment:
We can decouple this item api from the main application.

Reply:
Done! decoupled the routes for now. Later on, we can organize as apps (schemas, models and routes) all at the same folder.